tails_files/install.sh

```diff
@@ -41,11 +47,19 @@ chown root:root $ADDITIONS
 chmod 400 $ADDITIONS
 chown amnesia:amnesia $INSTALL_DIR/securedrop_icon.png
 chmod 600 $INSTALL_DIR/securedrop_icon.png
-chown amnesia:amnesia $INSTALL_DIR/.xsessionrc
-chmod 700 $INSTALL_DIR/.xsessionrc
+chown root:root $INSTALL_DIR/70-tor-reload.sh
```

Comment:
This requires more work, but since `70-tor-reload.sh` is run by the NetworkManager, is it run as root? If so, we can get rid of the setuid `securedrop_init` completely. Getting rid of setuid programs is generally a good idea for security.